src/prefect/workers/base.py

```diff
@@ -487,7 +514,10 @@ async def teardown(self, *exc_info):
 
     async def get_and_submit_flow_runs(self):
         runs_response = await self._get_scheduled_flow_runs()
+
         self._emit_worker_poll_flow_run_event()
+        self._last_polled_time = pendulum.now("utc")
```

Comment:
so we'll set this new `_last_polled_time` property on every interval...and then a call to `GET /health` will check how long its been since we last recorded a flow run poll

note that this omits cancellation checks - we can throw this attribute setting in there as well, but i figured scheduled flow runs are more important?